### ISSUE TITLE
Question on the terminology that is used in your paper

### ISSUE THREAD

**Issue by yoonguusong** [cwmok/LapIRN#10]
Hello.

congratulation on the first place in MICCAI registration challenge.

Because your paper is famous, I was reading your paper from this one.

However, I was curious the terminology that you used in this paper.

Of course these terminology was used not only in your paper, as well as other paper, 

but I was curious what is difference between 

displacement vector field, registration field, dense vector field, deformation field.

Even though there are many papers use these terminology, but I thought those are similar one only has different name.

but in your paper you specially use dense vector field, and deformation field different paragraph.

if it is fine, could you share your time to explain what is the difference between these terminology?

from the "An Unsupervise Learning Model for Deformable Medical Image Registration" which is preliminary study of voxelmorph, 

using 3 channel from the Unet output, I thought that is the phi (registration field, which is called as displacement vector field).
![dvf](https://user-images.githubusercontent.com/43869535/142456771-bf601e64-44ef-4d3f-9580-1bd5484de0e1.PNG)

but I cannot understand the what is difference between dense vector field  and deformation field in your paper..

could you explain it if you have time.

I really curious about this, so I thought this is the best way to ask you directly.

Sorry for bothering you.

Thanks

**Comment by cwmok** [comment]
Hi @yoonguusong,

> congratulation on the first place in MICCAI registration challenge.

Thanks. :D

> what is the difference between displacement vector field, registration field, dense vector field, deformation field.

To my understanding, The "deformation field" and "registration field" refer to the same/similar thing, which is a generalized term to describe the non-linear transformation grid (or flow-field grid) that is used to warp the moving image. For example, the `grid_sample` function in [Pytorch](https://pytorch.org/docs/stable/generated/torch.nn.functional.grid_sample.html) takes any grid and moving image as input and outputs the warped moving image using the input grid. The grid can be parameterized with any transformation model, like B-spline, displacement vector field, stationary vector field, affine transformation etc. 

The "displacement vector field", "dense vector field" and "displacement field" refer to a more specific transformation model that is defined as \phi(x) = x + u(x), where x is an identity transform and u is the displacement vector field.  This is the simplest transformation model in non-linear image registration methods and doesn't guarantee any diffeomorphic property. I suggest you read a classic paper, "_A fast diffeomorphic image registration algorithm_" by John Ashburner. It explains many terms used in image registration systematically.

>  From the "An Unsupervise Learning Model for Deformable Medical Image Registration" which is preliminary study of voxelmorph, using 3 channel from the Unet output, I thought that is the phi (registration field, which is called as displacement vector field).

You're correct. It outputs the 3-channel displacement field for 3D image registration. The 3-channel displacement field defines the displacement in the x, y and z-axis.

**Comment by cwmok** [comment]
Hi @yoonguusong,

> congratulation on the first place in MICCAI registration challenge.

Thanks. :D

> what is the difference between displacement vector field, registration field, dense vector field, deformation field.

To my understanding, The "deformation field" and "registration field" refer to the same/similar thing, which is a generalized term to describe the non-linear transformation grid (or flow-field grid) that is used to warp the moving image. For example, the `grid_sample` function in [Pytorch](https://pytorch.org/docs/stable/generated/torch.nn.functional.grid_sample.html) takes any grid and moving image as input and outputs the warped moving image using the input grid. The grid can be parameterized with any transformation model, like B-spline, displacement vector field, stationary vector field, affine transformation etc. 

The "displacement vector field", "dense vector field" and "displacement field" refer to a more specific transformation model that is defined as \phi(x) = x + u(x), where x is an identity transform and u is the displacement vector field.  This is the simplest transformation model in non-linear image registration methods and doesn't guarantee any diffeomorphic property. I suggest you read a classic paper, "_A fast diffeomorphic image registration algorithm_" by John Ashburner. It explains many terms used in image registration systematically.

>  From the "An Unsupervise Learning Model for Deformable Medical Image Registration" which is preliminary study of voxelmorph, using 3 channel from the Unet output, I thought that is the phi (registration field, which is called as displacement vector field).

You're correct. It outputs the 3-channel displacement field for 3D image registration. The 3-channel displacement field defines the displacement in the x, y and z-axis.

**Comment by yoonguusong** [comment]
thank you for letting me have your honorable time.
gratitude on your explanation. Thanks.
